tensorflow/python/custom_zeta/custom_zeta

```diff
@@ -0,0 +1,14 @@
+# custom_zeta.py
+import tensorflow as tf
+import mpmath
```

Comment:
We should not use the mpmath library - we don't want the dependency.  If people want, they can always do this in their own code.